### PR TITLE
Include completion change to make submit atomic.

### PR DESCRIPTION
**Pull request by jcdyer**
Include work to make completion atomic again.

https://github.com/edx/completion/pull/28

Reviewer:

- [ ] @bradenmacdonald / @pomegranited 

Deadline: ASAP

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @jcdyer! I've created [OSPR-2601](https://openedx.atlassian.net/browse/OSPR-2601) to keep track of it in JIRA. JIRA is a place for product owners to prioritize feature reviews by the engineering development teams.

Feel free to add as much of the following information to the ticket:
- supporting documentation
- edx-code email threads
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will still be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).

File: requirements/edx/base.txt [hunk 1]
```diff
@@ -219,4 +219,4 @@ edx-sga==0.6.2
 redis==2.10.6
 
 # Completion
-edx-completion==0.1.7
+git+https://github.com/edx/completion@cliff/atomic-submit#egg=edx-completion==0.1.8a1
```
Comment (pomegranited) [inline requirements/edx/base.txt]: My only worry here is that when https://github.com/edx/completion/pull/28 merged, this branch could be deleted?  So instead, could you push your branch to a name that will be preserved, and use the commit hash instead?

Comment (jcdyer) [reply requirements/edx/base.txt]: Makes sense.  Tagging as `oc-0.1.9-1`.

**Comment by pomegranited** [comment]
:+1: @jcdyer 

- [x] I tested this on my solutions devstack, by installing the updated dependency, and ensuring that the change it represents made it into the virtualenv.
- [x] I read through the code
- [x] <strike>I checked for accessibility issues</strike> N/A
- [x] <strike>Includes documentation</strike> N/A

**Comment by wajeeha-khalid** [comment]
@jcdyer Can you please let me know which JIRA board ticket this PR belongs too? i would need either MCKIN or YONK board ticket to add this in release notes.

**Comment by bradenmacdonald** [comment]
@wajeeha-khalid  YONK-1111

**Pull request merged**